resume/src/components/Basics.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import Profiles from './Profiles'
+import '../App.css';
+
+function Basics(props){
+    let basics = props.data
+    return(
+        <div className="basics">
+            <img src={basics.picture}></img>
+            <h1>{basics.name}</h1>
+            <small>{basics.label}</small>
+            <hr/>
+            <p className='contact'>{basics.email}</p>
+            <p className='contact'>{basics.phone}</p>
+            <p className='contact'>{basics.website}</p>
```

Comment:
I would make this an `a` tag if it is holding an actual website link for readability and accessibility!